### PR TITLE
✨ Frontend: Select resources & Credits workflow

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This set of features/UI elements are only available in master and for the s4l product. For now, it only lives in the frontend, but we want to bring it in to start getting some UX feedback.

When starting a new project from the study browser or service browser, a pop up window will be shown where the user will need to select in which machine the service should run. Depending on the selection, more or less credits will be charged.

This PR also implements the "Buy Credits" dummy workflow.

Select resources:
Resources

Credits workflow:
Credits

## Related issue/s

https://github.com/ITISFoundation/osparc-issues/issues/1027
closes https://github.com/ITISFoundation/osparc-simcore/pull/4497

### CODE REVIEW

**Comment by mguidon** [comment]
Its a start. We might have other resources and there might be a weight for simulation hours depending on whether you use a GPU of CPU instance. But this is still not 100 % clear

**Review by YuryHrytsuk — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/dashboard/ServiceBrowser.js [hunk 1]
```diff
@@ -112,23 +112,57 @@ qx.Class.define("osparc.dashboard.ServiceBrowser", {
...
       this.resetSelection();
     },
 
-    _createStudyFromService: function(key, version) {
+    _createStudyFromService: async function(key, version) {
       if (!this._checkLoggedIn()) {
         return;
       }
 
+      const isDevel = osparc.utils.Utils.isDevelopmentPlatform();
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/dashboard/ServiceBrowser.js:125]: @matusdrobuliak66 all API entries connected to this functionality should be enabled/disabled using `WEBSERVER_DEV_FEATURES_ENABLED`

**Review by pcrespov — COMMENTED** [review]
:tada: cool!

**Review by elisabettai — APPROVED** [review]
Very nice!

File: services/static-webserver/client/source/class/osparc/dashboard/ServiceBrowser.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/dashboard/ServiceBrowser.js:125]: @matusdrobuliak66 Notified 👌

**Review by GitHK — COMMENTED** [review]
To not forget.
As a user I would like to have the opportunity of customising and picking the sizes for each of my service in the project.
I would call such a feature an advanced or expert mode.
It would also be very good to have a spec which are what we propose an "suggested" resources (some balanced good performance profile selection) and then have options for running with maximum and minimum specs via dedicated buttons.

**Pull request merged**